python/chigger/tests/speed/speed.py

```diff
@@ -12,8 +12,13 @@
 import os
 import time
 import chigger
+import mooseutils
 
-filename = os.path.join(os.getenv('MOOSE_DIR'), 'modules', 'phase_field', 'examples', 'grain_growth', 'grain_growth_2D_graintracker_out.e')
+if os.getenv('MOOSE_DIR') == None:
+    mooseutils.mooseError("MOOSE_DIR must be set in this script")
+
+# filename = os.path.join(os.getenv('MOOSE_DIR'), 'modules', 'phase_field', 'examples', 'grain_growth', 'grain_growth_2D_graintracker_out.e')
+filename = os.path.join(os.getenv('MOOSE_DIR'), 'modules', 'phase_field', 'test', 'tests', 'grain_growth', 'gold', 'test_out.e-s005')
```

Comment:
Why this change, and did you forget to remove the commented out previous line?

Reply:
the original input is not included in the repository. it only appears after running the test suite for phase field. I ll remove the previous line

Reply:
Removed the previous line and a few commented lines in the input